ostelco-ios-client/ModelControllers/UserManager.swift

```diff
@@ -63,75 +63,6 @@ class UserManager: TokenProvider {
         OnBoardingManager.sharedInstance.region = nil
     }
     
-    func getDestinationFromContext() -> Promise<PostLoginDestination> {
```

Comment:
Let it burn! 🔥

Reply:
It's not burnt, it's just broken into pieces 😄

Reply:
Close enough!